requirements.txt

```
@@ -1,6 +1,7 @@
 setuptools
-numpy>=1.23.2
+numpy=1.23.2
```

```suggestion
numpy>=1.23.2,<2
```
The single `=` syntax broke the install tests. Will this work to avoid NumPy 2?

---

As long as it doesn't install numpy 2 it will be fine. This doesn't install numpy 2 when I use it, and the rest of the numpy version should be compatible. I will push with this change and see if it fixes it.

---

Seems like the tests need raw=True but other than that, your suggestion seems to be working okay and is not installing numpy 2 so I will mark this as resolved